<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTLVClientDlg</class>
 <widget class="QDialog" name="TTLVClientDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send message</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Client priKey</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="mPortText">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mHostText"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Client cert</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>KMIP Information</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>CA cert</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="mResponseText"/>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="mSendBtn">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="mViewResponseBtn">
     <property name="text">
      <string>View Response</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="mCloseBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Response Message</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="mCACertPathText"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="mClientCertPathText"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="mClientPriKeyPathText"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="mFindCABtn">
     <property name="text">
      <string>Find CA</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="mFindCertBtn">
     <property name="text">
      <string>Find Cert</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="mFindPriKeyBtn">
     <property name="text">
      <string>Find PrivateKey</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
